JavaScript/package-lock.json

```diff
@@ -12,34 +12,34 @@
 			"resolved": "https://registry.npmjs.org/@ava/babel-preset-stage-4/-/babel-preset-stage-4-1.1.0.tgz",
 			"integrity": "sha1-rmC+iBoLq/fTX1Krp3DR9hlPdr0=",
 			"requires": {
-				"babel-plugin-check-es2015-constants": "^6.8.0",
```

Comment:
Can you revert these changes? We should only bump dependencies if really necessary for some new functionality or security fix.